src/runtime/interface-info.ts

```diff
@@ -10,6 +10,7 @@
 
 import {assert} from '../platform/assert-web.js';
 
+import {Predicate} from '../runtime/hot.js';
 import {TypeChecker} from './recipe/type-checker.js';
 import {Type, TypeVariable, TypeLiteral} from './type.js';
```

Comment:
Should imports be alphabetical?

Reply:
there was a lint fixer but we encountered errors.  Will likely revisit as we move from tslint -> eslint.